PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java

```diff
@@ -79,9 +115,9 @@ private void sendCheckoutRequest(final FragmentActivity activity, final PayPalNa
 
         braintreeClient.getConfiguration((configuration, error) -> {
             sendPayPalRequest(
-                activity,
-                payPalCheckoutRequest,
-                configuration
+                    activity,
```

Comment:
Is the spacing here and below for the rest of the diff on this file correct? It looks really far in now but that may just be my Android n00b eyes.

Reply:
The linter wilds out sometimes lol. I was able to reduce this to one line.